sensing/preprocessor/gnss/gnss_poser/src/gnss_poser_core.cpp

```diff
@@ -351,4 +339,4 @@ void GNSSPoser::publishTF(
   tf2_broadcaster_.sendTransform(transform_stamped);
 }
 
-}  // namespace GNSSPoser
+RCLCPP_COMPONENTS_REGISTER_NODE(GNSSPoser)
```

Comment:
I like this pattern to get of the hand-written main function!

Reply:
It is a lot nicer :) Since it's already here, no need to revert, but didn't we agree in the last standup to keep the component porting for later?